rj_gameplay/rj_gameplay/play/wall_ball.py

```diff
@@ -9,11 +9,11 @@
 from typing import Dict, Generic, Iterator, List, Optional, Tuple, Type, TypeVar
 import numpy as np
 
+
```

Comment:
this file is identical to the file merged into ros2 recently by #1661, shows here bc I ran `make pretty-lines`